.github/workflows/py-ci.yml

```diff
@@ -185,6 +185,8 @@ jobs:
           # from thumbnail lambda fails whithout installed wheel.
           # See https://github.com/pypa/pip/issues/8559.
           python -m pip install wheel
+          # HACK: Pre-install numpy v1 as a build dependency for tifffile to prevent it from using v2 and failing to build
+          python -m pip install 'numpy<2'
```

Comment:
I'd put this under `if` similar to the one below.

Reply:
ok. btw i'm not quite sure why that `if [ ... ] ... then ... else ... fi` statement has something between if and then? is it a bug?